packages/larva-patterns/modules/story/story.right.js

```diff
@@ -0,0 +1,19 @@
+const clonedeep = require( 'lodash.clonedeep' );
+
+const story_right = clonedeep( require( './story.prototype' ) );
+
+const horizontal_alignment = ' lrv-u-justify-content-end';
+
+story_right.story_grid_primary_classes = story_right.story_grid_primary_classes.replace(
```

Comment:
Copy/paste to reassign the entire value with the replaced string - no `.replace`. Allowing the use of array and string manipulation methods resulted in the incredible maze that is known as the VY code-base. It's redundant, yes, but definitely preferable to that.